lib/atrium/institution.rb

```diff
@@ -17,21 +16,12 @@ def self.credentials(institution_code)
       end
     end
 
-    def self.list(options = nil)
-      endpoint = format_endpoint("institutions", options)
-      response = ::Atrium.client.make_request(:get, endpoint)
-
-      response["institutions"].map do |institution|
-        ::Atrium::Institution.new(institution)
-      end
+    def self.list(query_params: nil, limit: nil )
```

Comment:
nit picky but whitespace in the method. should be `def self.list(query_params: nil, limit: nil)`